### ISSUE TITLE
Readd back in VB support

### ISSUE THREAD

**Issue by crutkas** [Microsoft/WindowsTemplateStudio#1091]
To ship 1.3, we had to remove VB support as it would show up as an option.  After we ship, we need to add this back in ASAP & bonus points if we can make this configurable in case VB does not ship with 1.4

https://github.com/Microsoft/WindowsTemplateStudio/pull/1088

**Comment by mrlacey** [comment]
It shouldn't have been there anyway.
It was removed from the VB branch (that got merged into dev) in https://github.com/Microsoft/WindowsTemplateStudio/pull/864/commits/5369873d2ec01cea4921dd6405442cc79811ec7c

The history for the vsixmanifest file shows it's not there https://github.com/Microsoft/WindowsTemplateStudio/commits/5369873d2ec01cea4921dd6405442cc79811ec7c/code/src/Installer.2017/source.extension.vsixmanifest


The original commit message for VB support makes note that this has been removed
>This includes the VSIX template for VB support but deliberately does not include it in the installer. (We don't want to expose this until the templates are ready for use.)

The first [commit](https://github.com/Microsoft/WindowsTemplateStudio/commit/52bd6db372a72a427aa367dac835adb54f1845ea) to the branch (created from dev & kept in sync with it) where VB templates are being developed adds this functionality as it wasn't enabled.

And the answer:
It was added (incorrectly) back in as part of https://github.com/Microsoft/WindowsTemplateStudio/commit/3de2060ee8a544f0ebd1328f9a838758aca5f4f3
It's hard to follow the full history of what happened here but it looks like it was accidentally included as part of another change. Probably a branch being used for multiple things at once and multiple tests/features being accidentally committed together.

**Follow up:** As changes to the installer are potentially more serious (such as, like here, changing what gets installed) should they have extra process requirements for reviewing them?

**Comment by mrlacey** [comment]
Closing this as it's based on the false assumption that the VB references should have been there in the first place.

The VB reference in the installer should only be enabled in the installed when #863 is complete. The [branch where work is being done](https://github.com/Microsoft/WindowsTemplateStudio/tree/Issue863-VBTemplates) for #863 already includes [the change to add the reference to the installer](https://github.com/Microsoft/WindowsTemplateStudio/commit/52bd6db372a72a427aa367dac835adb54f1845ea#diff-d4fe12e36a33bea656d2d33ad50bce9d).


Even though closing, the follow-up question from my last comment still stands.

**Comment by ralarcon** [comment]
@mrlacey, It was me reviewing the manifest as I was thinking... "I cant access VB from the installer... here is"
